packages/odata-common/src/entity-base.ts

```diff
@@ -260,6 +260,14 @@ export abstract class EntityBase {
       : names.filter(key => !equal(this.remoteState[key], currentState[key]));
   }
 
+  /**
+   * Overwrites the default toJSON method so that all instance variables as well as all custom fields of the entity are returned.
```

Comment:
I guess this is new function to be shared.
"Overwrites" seems to be a copy paste issue.

Reply:
I copied it from the concrete entity: https://github.com/SAP/cloud-sdk-js/blob/f43da96fb9efbc5297f4279fa80060d542830c97/test-packages/test-services/v2/test-service/TestEntity.ts#L172